modules/ding_frontend/ding_frontend.admin.inc

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * @file
+ * Module's admin features.
+ */
+
+/**
+ * Administration form.
+ */
+function ding_frontend_settings($form, &$form_state) {
+  $types = array(
+    'ding_news' => t('News'),
+    'ding_event' => t('Event'),
+  );
+  $form['ding_frontend_fe_promote_types'] = array(
+    '#title' => t('Promote types to Frontpage'),
+    '#type' => 'checkboxes',
+    '#options' => $types,
+    '#default_value' => variable_get('ding_frontend_fe_promote_types', array('ding_event' => 'ding_event')),
```

Comment:
I have not had a chance to finish the review yet but in [one of my earlier comments](https://github.com/ding2/ding2/pull/74#discussion_r46824193) I suggested:

> removing the variables from the corresponding features and instead setting their defaults in a hook_install() in their respective modules